src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -58,68 +57,65 @@
 	public void robotInit() {
 		// Initialize logging framework
 		Logging.init();
-		
+
 		// Initialize RobotMap
 		RobotMap.init(RobotID.Competition_1);
 		
 		// Make robot objects
-		elevator = Elevator.getInstance();
 		driverstation = DriverStation.getInstance();
 		LOGGER.info("Initialized Driverstation");
-		
+
 		drive = Drive.getInstance();
+
+		gyro = Gyrometer.getInstance();
+		gyro.calibrate();
+		gyro.reset();
 		
-		//gyro = Gyrometer.getInstance();
 		grabber = Grabber.getInstance();
-		
-		//gyro.calibrate();
-		//gyro.reset();
-		
 		elevator = Elevator.getInstance();
-
+		
 		// Initialize math lookup table
 		LookUpTable.init();
-		
-//		vision = VisionProcessing.getInstance();
-		
+
+		//		vision = VisionProcessing.getInstance();
+
 		// TODO: Implement actions.doNothing
-//		autonomous = Actions.doNothing();
-		
+		//		autonomous = Actions.doNothing();
+
 		//made usb camera and captures video
-		//UsbCamera cam = CameraServer.getInstance().startAutomaticCapture();
+		UsbCamera cam = CameraServer.getInstance().startAutomaticCapture();
 		//set resolution and frames per second to match driverstation
-		//cam.setResolution(320, 240);
-		//cam.setFPS(15);
-		
+		cam.setResolution(320, 240);
+		cam.setFPS(15);
```

Comment:
I'd rather leave the camera commented out for now. Otherwise we crash looking for cameras that don't exist.